code/lib/cli/src/repro-templates.ts

```diff
@@ -67,6 +67,20 @@ const vue3ViteTemplates = {
   },
 };
 
+const vue2ViteTemplates = {
+  'vue2-vite/2.7-js': {
+    name: 'Vue2 Vite (vue 2.7 JS)',
+    script:
+      'yarn create vite . --template vanilla && yarn add --dev @vitejs/plugin-vue2 vue-template-compiler vue@2 && echo "import vue2 from \'@vitejs/plugin-vue2\';\n\nexport default {\n\tplugins: [vue2()]\n};" > vite.config.js',
```

Comment:
@ndelangen WDYT about this? we said we were going to only use community bootstrap tools for this. do we use this as a placeholder for now & then move to some storybook package later?

Reply:
The challenge is, there's no generator / bootstrap for using Vue 2 with Vite, as far as I could find.  Vue 2 had its own cli, and it's necessary to create a Vue 2 project and then manually convert it over to Vite.  Or, this approach which required fewer changes, which is to add a vanilla Vite project, and then add Vue2 into it.

Reply:
understood! thank you for the explanation, let's do a follow up at some point to extract this into the public domain in some sort of CLI project generator.